plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/reporting/VerifierErrorInterpreter.kt

```diff
@@ -37,6 +39,12 @@ class VerifierErrorInterpreter {
                 reportOn(source, PluginErrors.UNEXPECTED_RETURNED_VALUE, "null", context)
             error is PostconditionViolated && role is SourceRole.CallsInPlaceEffect ->
                 reportOn(source, PluginErrors.INVALID_INVOCATION_TYPE, role.paramSymbol, role.kind.asUserFriendlyMessage, context)
+            (error is PreconditionInCallFalse || error is AssertFailed) && role is SourceRole.ParamFunctionLeakageCheck -> {
+                val paramSymbol = error.extractInfoFromFunctionArgument(role.INDEX).unwrapOr<FirBasedSymbol<*>> {
+                    error("The function symbol is missing.")
```

Comment:
Please provide a little more info here.  The function symbol of what?

Reply:
I will remove the contained error message, since I'm going to define an `unwrap` [function](https://github.com/jesyspa/kotlin/pull/184#discussion_r1389133505) to avoid the bang-bang operator.